Unity-Technologies/ScriptableRenderPipeline
# Move LWRP tests out of package and into project

### Purpose of this PR
Moving LWRP tests from com.unity.testing.srp.lightweight to the project in TestProjects to make the tests consistent with other tests in the repo.

---
### Release Notes
Moving LWRP tests from com.unity.testing.srp.lightweight to the project in TestProjects to make the tests consistent with other tests in the repo.

---
### Testing status
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=move-LW-tests-into-project&unity_branch=trunk&automation-tools_branch=master
The tests ran, and the failures are known issues (and prove that the tests did run)

**Manual Tests**:
The tests have been run locally to confirm that they still appear and run in the Test Runner window

**Automated Tests**:
No tests added or removed - they were moved from one place to another.

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
None

**Halo Effect**: None, Low, Medium, High?
None
